net-analyzer/nmap/nmap-7.92-r1.ebuild

```
@@ -29,7 +29,7 @@ else
 fi
 
 SLOT="0"
-IUSE="ipv6 libssh2 ncat nping +nse ssl +system-lua"
+IUSE="ipv6 libssh2 ncat nping +nse ssl symlink +system-lua"
 REQUIRED_USE="system-lua? ( nse ${LUA_REQUIRED_USE} )"
```

symlink only affects USE=ncat, so...
```suggestion
REQUIRED_USE="system-lua? ( nse ${LUA_REQUIRED_USE} )
	symlink? ( ncat )"
```

---

ah nice, I forgot this. It avoids to do the check later.

---

ah nice, I forgot this. It avoids to do the check later.